spring-integration-ip/src/main/java/org/springframework/integration/ip/config/IpAdapterParserUtils.java

```diff
@@ -183,8 +208,8 @@ public static void addPortToConstructor(Element element,
 	static String getPort(Element element, ParserContext parserContext) {
 		String port = element.getAttribute(IpAdapterParserUtils.PORT);
 		if (!StringUtils.hasText(port)) {
-			parserContext.getReaderContext().error(IpAdapterParserUtils.PORT +
-					" is required for IP channel adapters", element);
+			parserContext.getReaderContext().error(IpAdapterParserUtils.PORT
+					+ " is required for IP outbound channel adapters", element);
```

Comment:
This method is used from the `UdpInboundChannelAdapterParser` as well.
Therefore `outbound` word here isn't correct.
The change should be reverted.